pyproject.toml

```diff
@@ -1,46 +1,57 @@
 [build-system]
-requires = ["setuptools"]
-build-backend = "setuptools.build_meta"
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
 
-[project]
+[tool.poetry]
 name = "trtools"
 version = "5.1.1"
 authors = [
-    {name = "Melissa Gymrek", email = "mgymrek@ucsd.edu"},
-    {name = "Gymrek Lab"},
+    "Melissa Gymrek <mgymrek@ucsd.edu>",
+    "Gymrek Lab",
 ]
 description = "Toolkit for genome-wide analysis of STRs"
 readme = "README.rst"
-requires-python = ">=3.6"
-license = {text = "MIT"}
+license = "MIT"
 classifiers = [
-    "Programming Language :: Python :: 3.5",
-    "License :: OSI Approved :: MIT License",
     "Operating System :: OS Independent",
     "Intended Audience :: Science/Research",
     "Topic :: Scientific/Engineering :: Bio-Informatics",
 ]
-dependencies = [
-    "importlib-metadata", # required as long as we support py<3.8
-    "cyvcf2",
-    "matplotlib",
-    "numpy",
-    "pandas",
-    "pysam",
-    "scikit-learn",
-    "scipy",
-    "statsmodels",
-    "pyfaidx",
+exclude = [
+    "trtools/testsupport/*",
+    "doc",
+    "example-files",
+    "trtools/*/*.rst",
+    "trtools/*/Example_Files",
+    "trtools/dumpSTR/filter_files",
```

Comment:
These are useful for running commands and shouldn't be excluded unless we replace them with documentation showing where to download them from.